addons/flannel/0.21.5/install.sh

```diff
@@ -270,9 +270,35 @@ function remove_weave() {
 
     # all nodes
     ip link delete weave
+
     rm -rf /var/lib/weave
     rm -rf /etc/cni/net.d/*weave*
     rm -rf /opt/cni/bin/weave*
+
+    logStep "Verifying Weave removal"
+    # Check if any Weave-related resources are still present
+    weave_resources=$(kubectl -n kube-system get pods --no-headers -l name=weave-net || true)
+
+    if [ -n "$weave_resources" ]; then
+        logWarn "Weave resources are still present. Deleting Weave resources with grace period up to 3 minutes..."
+        kubectl -n kube-system delete pods -l name=weave-net --grace-period=180
```

Comment:
@emosbaugh I need came back it for as it was.
See that in your suggestion you changed it for we delete ONLY pods when we have other kinds of resource and we should remove ALL. Because of this we need :

> kubectl -n kube-system delete $(kubectl get all -n kube-system | grep weave | awk '{print $1}')